frontend/lib/models/user.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:dartx/dartx.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'user.freezed.dart';
+part 'user.g.dart';
+
+@freezed
+class User with _$User {
+  const User._();
+
+  const factory User({
+    required int id,
+    required String name,
+    @JsonKey(name: 'launchpad_email') required String launchpadEmail,
+    @JsonKey(name: 'launchpad_handle') required String launchpadHandle,
+  }) = _User;
+
+  factory User.fromJson(Map<String, Object?> json) => _$UserFromJson(json);
+
+  String get initials {
+    final names = name.split(' ');
+    final numOfNames = names.length;
```

Comment:
could you please explain why we can have multiple names here?

Reply:
As in first name, middle name, last name

Reply:
cool, thanks!